drivers/google/google_test.go

```diff
@@ -0,0 +1,153 @@
+package google
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/codegangsta/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriverName(t *testing.T) {
+	driverName := newDriver().DriverName()
+
+	assert.Equal(t, "google", driverName)
+}
+
+func TestFailWithoutProject(t *testing.T) {
+	_, err := create([]string{"test-name"})
+
+	assert.EqualError(t, err, "Please specify the Google Cloud Project name using the option --google-project.")
+}
+
+func TestParseFlags(t *testing.T) {
+	driver, err := create([]string{
```

Comment:
But this issue seems to be due to the fact that the tags flag was set as `String`, not as `StringSlice`: https://github.com/docker/machine/blob/master/drivers/google/google.go#L104

`codegangsta/cli` seems to be rightfully complaining that it received a `StringFlag` for that `google-tags` value and in `SetConfigFromFlags` you are trying to query it as a `StringSlice`.